src/Service/CloudWatchLogs/tests/Unit/Result/DescribeLogStreamsResponseTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace AsyncAws\CloudWatchLogs\Tests\Unit\Result;
+
+use AsyncAws\CloudWatchLogs\Result\DescribeLogStreamsResponse;
+use AsyncAws\CloudWatchLogs\ValueObject\LogStream;
+use AsyncAws\Core\Response;
+use AsyncAws\Core\Test\Http\SimpleMockedResponse;
+use AsyncAws\Core\Test\TestCase;
+use Psr\Log\NullLogger;
+use Symfony\Component\HttpClient\MockHttpClient;
+
+class DescribeLogStreamsResponseTest extends TestCase
+{
+    public function testDescribeLogStreamsResponse(): void
+    {
+        // see https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_DescribeLogStreams.html
+        $response = new SimpleMockedResponse('{
+           "logStreams": [
+              {
+                 "arn": "test_arn",
```

Comment:
same here (use AWS code snippet).

Reply:
I'll update that, by the way, the sample response on https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_DescribeLogStreams.html#API_DescribeLogStreams_Examples is wrong, storageBytes should be storedBytes

also, as explained in https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_LogStream.html#API_LogStream_Contents the value of storedBytes is always zero